File: src/main/java/app/PruebaCazadores.java
package app;

import dominio.personaje.Alita;
import dominio.personaje.Buffy;
import dominio.interfaces.Cazador;

import java.util.Arrays;
import java.util.List;

public class PruebaCazadores {
    public static void main(String[] args) {
        Buffy buffy = new Buffy();
        Alita alita = new Alita("espada");
//        List cazadores = Arrays.asList(buffy, alita);
//        salirACazar(cazadores);
    }

//    private static void salirACazar(List cazadores) {
//        cazadores.forEach(cazador -> cazador.cazar());
//    }
}

File: app/prueba_cazadores.js
import {Alita} from './dominio/personajes/Alita.js';
import {Buffy} from './dominio/personajes/Buffy.js';

// Cazadores 
const alita = new Alita("espada");
const buffy = new Buffy();

const cazadores = [alita, buffy];

cazadores.forEach(cazador => cazador.cazar());
